accord-core/src/test/java/accord/topology/TopologyUtils.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package accord.topology;
+
+import accord.api.RoutingKey;
+import accord.impl.IntKey;
+import accord.impl.PrefixedIntHashKey;
+import accord.primitives.Range;
+import accord.local.Node;
+import accord.primitives.Ranges;
+import accord.primitives.RoutingKeys;
+import accord.primitives.Unseekables;
+import accord.primitives.Unseekables.UnseekablesKind;
+import accord.utils.Gens;
+import accord.utils.RandomSource;
+import accord.utils.Utils;
+import accord.utils.WrapAroundList;
+import accord.utils.WrapAroundSet;
+
+import javax.annotation.Nullable;
+import java.util.*;
+
+import static accord.utils.Utils.toArray;
```

Comment:
this was moved into `accord.topology` package to get access to package private state (for `withEpoch`); most of the class is not new but git/GH doesn't seem to detect the move